backend/pkg/sqlconnect/sql-connector.go

```diff
@@ -427,7 +427,56 @@ func getGeneralDbConnectionConfigFromMysql(config *mgmtv1alpha1.ConnectionConfig
 			QueryParams: query,
 		}, nil
 	case *mgmtv1alpha1.MysqlConnectionConfig_Url:
-		return nil, nucleuserrors.NewNotImplemented("not currently implemented")
+		// follows the format [scheme://][user[:password]@]<host[:port]|socket>[/schema][?option=value&option=value...]
```

Comment:
If you could add some tests in the sql-connector_Test file that thoroughly test this branch with various mysql connection strings, that would be great! :) There are some postgres equivalent tests that you can example from

Reply:
sure thing